odoo/addons/base/models/ir_http.py

```diff
@@ -201,6 +201,24 @@ def _match(cls, path_info):
     def _get_public_users(cls):
         return [request.env['ir.model.data']._xmlid_to_res_model_res_id('base.public_user')[1]]
 
+    @classmethod
+    def _auth_method_bearer(cls):
+        uid = request.env.uid
+        if (
+            (authorization := request.httprequest.authorization)
+            and authorization.type == 'bearer'
+        ):
+            # werkzeug>=2.3 has a token attribute, but previously it was a parameter
+            # we support >=2.0, so we try both methods
+            token = getattr(authorization, 'token', authorization.get('token'))
```

Comment:
does `authorization.get('token')` works in 2.3? If so I would suggest we always use that so we don't need to `getattr`

Reply:
`get` is just an alias for `parameters.get`. Since 2.3, the attribute token is extracted from the parameters which become empty.